src/garage/sampler/__init__.py

```diff
@@ -16,8 +17,19 @@
 from garage.sampler.worker_factory import WorkerFactory
 
 __all__ = [
-    'BatchSampler', 'Sampler', 'ISSampler', 'singleton_pool', 'LocalSampler',
-    'RaySampler', 'SamplerWorker', 'ParallelVecEnvExecutor', 'VecEnvExecutor',
-    'OffPolicyVectorizedSampler', 'OnPolicyVectorizedSampler', 'WorkerFactory',
-    'Worker', 'DefaultWorker'
+    'BatchSampler',
+    'Sampler',
+    'ISSampler',
+    'singleton_pool',
+    'LocalSampler',
+    'RaySampler',
+    'SamplerWorker',
+    'ParallelVecEnvExecutor',
+    'VecEnvExecutor',
+    'OffPolicyVectorizedSampler',
+    'OnPolicyVectorizedSampler',
+    'WorkerFactory',
+    'Worker',
+    'DefaultWorker',
+    'PEARLWorker',
```

Comment:
please put this class in `src/garage/torch/algos/pearl.py` -- since it is only used by the PEARL algorithm.

Its import path will be `from garage.torch.algos.pearl import PEARLWorker`